src/App.js

```diff
@@ -28,19 +32,53 @@ class App extends Component {
   }
 
   render() {
-    const { ticker, websocket } = this.props;
+    const {
+      pairs,
+      tickerLastUpdated,
+      websocket,
+      websocketOffline,
+    } = this.props;
     return (
       <div>
-        <p>{websocket.status}</p>
-        <p>{JSON.stringify(ticker)}</p>
+        <Navbar pairs={pairs} tickerLastUpdated={tickerLastUpdated} />
+        <div id="content">
+          <p>{websocket.status}</p>
+          <p>
+            {(!!tickerLastUpdated &&
```

Comment:
remove double negative?

Reply:
You're right. I was doing it to avoid accidental render, but the only falsy value this may have is `null`, which should not cause a render.